src/EditorFeatures/CSharpTest/ExtractMethod/ExtractMethodTests.cs

```diff
@@ -10920,56 +10939,25 @@ public async Task Hello()
                 {
                     var cancellationToken = CancellationToken.None;
 
-                    int i = await NewMethod(ref cancellationToken);
+                    (int i, cancellationToken) = await NewMethod(cancellationToken);
```

Comment:
previous code was illegal (can't pass a parameter by ref/value to an async method.

Reply:
Just wanted to be sure, the reason that `cancellationToken` is returned here is because it is used again later in this method.